InvenTree/InvenTree/ready.py

```diff
@@ -42,6 +41,9 @@ def canAppAccessDatabase(allow_test: bool = False, allow_plugins: bool = False):
         'mediarestore',
     ]
 
+    if not allow_shell:
```

Comment:
@SchrodingersGat what has this to do with search?

Reply:
Nothing, but during testing for search I found that some functions were blocked entirely if running from the shell.

Reply:
Ah ok, I was a bit surprised what effect it could have on search 👍